sats-dna/src/main/kotlin/com/sats/dna/components/SatsOutlinedTextField.kt

```diff
@@ -50,6 +56,57 @@ fun SatsOutlinedTextField(
     )
 }
 
+object SatsOutlinedTextFieldDefaults {
+    @Composable
+    fun colors(
+        textColor: Color = M2LocalContentColor.current.copy(M2LocalContentAlpha.current),
+        disabledTextColor: Color = textColor.copy(M2ContentAlpha.disabled),
+        backgroundColor: Color = Color.Transparent,
+        cursorColor: Color = M2MaterialTheme.colors.primary,
+        errorCursorColor: Color = M2MaterialTheme.colors.error,
+        focusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.focused,
+        unfocusedBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        disabledBorderColor: Color = SatsTheme.colors2.graphicalElements.border.default,
+        errorBorderColor: Color = SatsTheme.colors2.graphicalElements.signal.error,
```

Comment:
Our own defaults, can be overridden if needed. Should we allow overriding these?

Reply:
I guess overriding if needed should be ok, if it's needed it's needed 🪡